cmd/incusd/api_cluster.go

```diff
@@ -3265,6 +3266,12 @@ func evacuateClusterMember(s *state.State, gateway *cluster.Gateway, r *http.Req
 			_ = evacuateClusterSetState(s, nodeName, db.ClusterMemberStateCreated)
 		})
 
+		 // TODO stop all networks here
```

Comment:
That should be done later, after `evacuateInstances` is done otherwise the instances are going to run into issues as the network disappears underneath them.